### PR TITLE
[DOC-7053] Transaction sub-types.

### PR DESCRIPTION
**Pull request by urbiz-nr**
Added table with transaction sub-types. See #2179 and DOC-7053.

### CODE REVIEW

File: src/content/docs/apm/transactions/intro-transactions/transactions-new-relic-apm.mdx [hunk 1]
```diff
@@ -29,6 +29,445 @@ Cumulative transaction data appears in APM on the [Transactions page](/docs/apm/
...
+    id="agent-go"
+    title="Go"
+  >
+  Doesn't apply.
+  </Collapser>
+  
+  <Collapser
+    id="agent-java"
+    title="Java"
+  >
```
Comment (x8a) [inline src/content/docs/apm/transactions/intro-transactions/transactions-new-relic-apm.mdx:86]: I think we're missing a `</table>` here

File: src/content/docs/apm/transactions/intro-transactions/transactions-new-relic-apm.mdx [hunk 2]
```diff
@@ -29,6 +29,445 @@ Cumulative transaction data appears in APM on the [Transactions page](/docs/apm/
...
+    <tr>
+      <td>
+        Function
+      </td>
+
+      <td>
+        The invocation of the named function.
+      </td>
+    </tr>
+  </Collapser>
```
Comment (x8a) [inline src/content/docs/apm/transactions/intro-transactions/transactions-new-relic-apm.mdx:74]: I think we're missing a `</table>` here.

Comment (urbiz-nr) [reply src/content/docs/apm/transactions/intro-transactions/transactions-new-relic-apm.mdx:76]: Sorry, I had forgotten to commit.

**Review by x8a — APPROVED** [review]
Looks great!

**Pull request merged**